src/components/TextField/tests/TextField.test.tsx

```diff
@@ -164,6 +191,16 @@ describe('<TextField />', () => {
       expect(type).toBe('email');
     });
 
+    it('sets the enableTextCouter on the input', () => {      
+      const enableTextCouter = mount(<TextField label="TextField" enableTextCouter={true} />).prop('enableTextCouter');
+      expect(typeof enableTextCouter).toBe('boolean');
+    });   
+
+    it('sets the maxLength on the input', () => {
+      const maxLength = mount(<TextField label="TextField" maxLength={100} />).prop('maxLength');      
+      expect(typeof maxLength).toBe('number');
+    });
+
```

Comment:
@tejashrigajjar We need additional unit tests to check the text of the counter thats rendered inside `{counterTextMarkup}` in the render() method.  To make sure, when there is no maxLength, only text length is rendered and when max length, then both are listed.